dvc/pkg/__init__.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import unicode_literals
+
+import os
+import copy
+import shutil
+
+from dvc.utils.compat import urlparse, str, builtin_str
+
+from dvc.config import Config
+from dvc.remote.config import remote_add
+
+from .config import get_pkg_settings
+
+
+class Pkg(object):
+    def __init__(self, repo, settings):
+        self.settings = copy.copy(settings)
+
+        name = None
+        if isinstance(settings, str) or isinstance(settings, builtin_str):
```

Comment:
qq: can we use one of them? str or builtin_str? if not can we move to compat as a helper?